core/templates/dev/head/pages/topics_and_skills_dashboard/skills_list_directive.html

```diff
@@ -1,82 +1,79 @@
-<div class="skills-list">
-  <md-card layout="row" class="list-view-item">
-    <table class="dashboard-table">
-        <colgroup>
-          <col style="width: 50%;">
-          <col style="width: 25%;">
-          <col style="width: 25%;">
-        </colgroup>
-      <tr>
-        <th ng-repeat="key in SKILL_HEADINGS"
-            class="dashboard-table-headings">
-          <p ng-if="key === 'description'" translate="I18N_DASHBOARD_TABLE_HEADING_SKILL_DESCRIPTION"></p>
-          <p ng-if="key === 'worked_examples_count'" translate="I18N_DASHBOARD_TABLE_HEADING_WORKED_EXAMPLES"></p>
-          <p ng-if="key === 'misconception_count'" translate="I18N_DASHBOARD_TABLE_HEADING_MISCONCEPTIONS"></p>
-        </th>
-      </tr>
-      <tr ng-repeat="skill in skillSummaries()"
-          class="list-item">
-        <td>
-          <a ng-href="<[getSkillEditorUrl(skill.id)]>" class="list-summary">
-            <span> <[skill.description]> </span>
-          </a>
-        </td>
-        <td>
-          <a ng-href="<[getSkillEditorUrl(skill.id)]>" class="list-summary">
-            <span> <[skill.worked_examples_count]> </span>
-          </a>
-        </td>
-        <td>
-          <a ng-href="<[getSkillEditorUrl(skill.id)]>" class="list-summary">
-            <span> <[skill.misconception_count]> </span>
-          </a>
-        </td>
-      </tr>
-    </table>
-  </md-card>
-</div>
+<md-card layout="row" class="list-view-item">
+  <table class="dashboard-table">
+    <colgroup>
+      <col style="width: 50%;">
+      <col style="width: 25%;">
+      <col style="width: 25%;">
+    </colgroup>
+    <tr>
+      <th ng-repeat="key in SKILL_HEADINGS"
+          class="dashboard-table-headings">
+        <p ng-if="key === 'description'"> Skill Description </p>
+        <p ng-if="key === 'worked_examples_count'"> Worked Examples </p>
+        <p ng-if="key === 'misconception_count'"> Misconceptions </p>
+      </th>
+    </tr>
+    <tr ng-repeat="skill in getSkillSummaries()"
+        class="list-item">
+      <td>
+        <a ng-href="<[getSkillEditorUrl(skill.id)]>" class="list-summary">
+          <span> <[skill.description]> </span>
+        </a>
+      </td>
+      <td>
+        <a ng-href="<[getSkillEditorUrl(skill.id)]>" class="list-summary">
+          <span> <[skill.worked_examples_count]> </span>
+        </a>
+      </td>
+      <td>
+        <a ng-href="<[getSkillEditorUrl(skill.id)]>" class="list-summary">
+          <span> <[skill.misconception_count]> </span>
+        </a>
+      </td>
+    </tr>
+  </table>
+</md-card>
 
 <style>
-  .skills-list .list-view-item {
+  skills-list .list-view-item {
```

Comment:
Just to check, why do you need the .list-view-item class at all -- wouldn't just "skills-list" suffice"?

Reply:
I tried by removing list-view-item, but then the styling was removed.